schemas/context/profile-adcloud.schema.json

```diff
@@ -0,0 +1,34 @@
+{
+  "meta:license": [
+    "Copyright 2017 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/context/profile-adcloud",
```

Comment:
In talking with the ad-cloud group, the purpose of this new class is because they didn't like the current Profile includes the identityMap be default. 
There has been talks to remove the identityMap from the class, so I would suggest we go that route and avoid an adcloud specific class within standard XDM.

Reply:
Agreed.  The identity map is optional anyway, so this should not be an issue.

Reply:
We can live with identity map for a short term unless it is removed from the class. Would removal of identity map be a breaking change for us - what I mean is do we have to re-patch our mix-in and schema at a later state? If not, we can live with what you have proposed above.

Reply:
We are planning to remove it from the class so you will not have it long. Should be going in about the same time as this PR. As long as you do not use that field for ingestion it will not be a breaking change.

What you would need to do is change the meta:intendedToExtend attribute to reference the core context/profile instead. Let's do that now as part of this PR.

Reply:
Removed Ad-Cloud Specific Record class and instead used base profile class to define this mix-in.